modules/odc-image-vulnerabilities-breakdown.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * applications/odc-monitoring-project-and-application-metrics-using-developer-perspective.adoc
+
+:_content-type: CONCEPT
+[id="odc-image-vulnerabilities-breakdown_{context}"]
+= Image vulnerabilities breakdown
+
+In the developer perspective, the project dashboard shows the *Image Vulnerabilities* link in the *Status* section. Using this link, you can view the *Image Vulnerabilities breakdown* pop-up that includes details regarding vulnerable container images and fixable container images. The severity level of vulnerability is highlighted using the following color icons:
+
+* Red: Indicates the vulnerability is of high priority for container image functioning and needs an immediate fix
+* Orange: Indicates the vulnerability is of medium priority and can be fixed after high priority vulnerabilities are fixed
+* Yellow: Indicates the vulnerability is of low priority and can be fixed after high and medium priority vulnerabilities are fixed
```

Comment:
```suggestion
* Red: High priority. Fix immediately.
* Orange: Medium priority. Can be fixed after high-priority vulnerabilities.
* Yellow: Low priority. Can be fixed after medium priority vulnerabilities.
```
Optional: Minimalism. I think you can remove redundant words and increase clarity.